osu.Game/Overlays/BeatmapSet/Scores/TopScoreStatisticsSection.cs

```diff
@@ -111,7 +111,8 @@ public ScoreInfo Score
 
                 accuracyColumn.Text = value.DisplayAccuracy;
                 maxComboColumn.Text = $@"{value.MaxCombo:N0}x";
-                ppColumn.Alpha = value.Beatmap?.Status == BeatmapSetOnlineStatus.Ranked ? 1 : 0;
+
+                ppColumn.Alpha = value.Beatmap?.Status.GrantsPerformancePoints() ?? false ? 1 : 0;
```

Comment:
```suggestion
                ppColumn.Alpha = value.Beatmap?.Status.GrantsPerformancePoints() == true ? 1 : 0;
```

Reply:
Can apply this, I do prefer the `== true` version slightly but I'm not overly fussed.